### ISSUE TITLE
Vector Layer does not get exported (but is respected for trimming)

### ISSUE THREAD

**Issue by ilegacy** [GDQuest/krita-batch-exporter#55]
- [X] Bug report.
- [ ] Feature request.

## Bug report ##

**What is the current behavior?**

Vector layer does not exported (is not visible) BUT trimming respects the bounding box of the vector text!

**What is the expected behavior?**

Vector layer is visible in exported PNG:

**Tell us the steps to reproduce the bug, and if possible share a minimal demo of the problem.** 

1. Get the file: https://www.dropbox.com/s/x8gddk0fnyw2bw8/gamejam_headlines_01.kra?dl=0
2. Click "All Layers" in the Batch Exporter
3. See that the file has the correct dimensions (after trimming) but the text is not readable.

**Comment by NathanLovato** [comment]
That's an unsupported feature, we don't support vector layers currently.

**Comment by ilegacy** [comment]
Oh, in that case I'm afraid the documentation is out of date: 
![Screenshot_20210814-105603_Brave](https://user-images.githubusercontent.com/12360416/129440753-ff91b880-12f9-4cae-bc9f-5a37789ab364.jpg)

**Comment by NathanLovato** [comment]
I checked the code, and indeed we do try to export vector layers. I didn't remember that, my bad. What I'm seeing too is we rely on Krita's internal conversion mechanism, and it seems it doesn't work with text perhaps?

@razcore-rad this user has a vector layer with text and the text doesn't get rasterized when exporting the corresponding layer with batch exporter. The output image is empty. Do you know something about that?

**Comment by razcore-rad** [comment]
@ilegacy It seems like there's a problem with non-default Color Profiles. After converting your image to the default profile:

![image](https://user-images.githubusercontent.com/1177508/129522735-1e76b6bb-360c-4f9e-aa88-a6f69f672a82.png)

I could successfully export the image:

![einleitung_01](https://user-images.githubusercontent.com/1177508/129522888-bb394b18-ed3f-434e-95e0-85bf52e4f656.png)

---

I don't know if this is an internal issue or a bug in our plugin yet.

**Comment by ilegacy** [comment]
@razcore-rad Thank you! Yes,  you are right. When I change the color-profile to default, save the file, re-load the file and re-export with the batch exporter, it works!

Thank you for the help!

**Comment by razcore-rad** [comment]
Yeah, I just don't know why this is happening, cause before save we convert the image in the plugin to the default color profile as well.
